source/nodejs/adaptivecards-designer/src/app.ts

```diff
@@ -212,8 +233,14 @@ class DesignerApp {
                     }
                 );
             }
-        
-            this.propertySheetHostElement.appendChild(card.render());
+
+            properties.appendChild(card.render());
+            let cardNodes = card.renderedElement.children;
+            (cardNodes[0] as HTMLElement).style.backgroundColor = "transparent";
```

Comment:
Please don't do that. If you want to control the colors, change the card's HostConfig or the associated CSS file, but don't patch the rendered DOM elements after the fact.

Reply:
P0: More override examples - use what adaptive offers and don't override